LICENSE

```diff
@@ -1,6 +1,6 @@
 nsx-container-plugin-operator
 
-Copyright (c) 2020 VMware, Inc.  All rights reserved.				
+Copyright (c) 2021 VMware, Inc.  All rights reserved.				

```

Comment:
I'm not sure if this needs to be updated for each release, usually we don't. But I don't think it'll do any harm updating it.

Reply:
Yes, I'm also not very sure on this update.  I did it as my previous experience.  Thanks for comments.